Cargo.toml

```diff
@@ -20,10 +20,11 @@ build = "build.rs"
 arc-swap = "1.5"
 async-trait = { version = "0.1.42", optional = true }
 bitflags = "1.1"
+snapshot = { git = "https://github.com/firecracker-microvm/firecracker", tag = "v1.4.1", optional = true }
```

Comment:
We need a published crate here, otherwise we can't publish fuse-backend-rs anymore.

Reply:
Is there a published version of `snapshot`?

Reply:
@bergwolf no, I cannnot find the crate on crates.io .

Reply:
hmm, I've created https://github.com/firecracker-microvm/firecracker/issues/4162. Let's see what the firecracker maintainers think.

Reply:
Now that we have https://crates.io/crates/dbs-snapshot, could you use the published one?

Reply:
Yes. I'll update it later.

Reply:
> Yes. I'll update it later.

done.